apps/etterlatte-beregning/src/main/kotlin/avkorting/AvkortingTidligAlderspensjonService.kt

```diff
@@ -45,7 +46,10 @@ class AvkortingTidligAlderspensjonService(
                 }
 
                 BehandlingType.REVURDERING -> {
-                    val aarsoppgjoer = avkorting.aarsoppgjoer.single { it.aar == behandling.virkningstidspunkt?.dato?.year }
+                    val aarsoppgjoer =
+                        avkorting.aarsoppgjoer.single {
+                            it.aar == behandling.virkningstidspunkt?.dato?.year
+                        } as AarsoppgjoerLoepende
```

Comment:
Vet ikke om det er mulig å ta med en `is AarsoppgjoerLoepende` evt sjekke via `as? AarsoppgjoerLoepende ?: throw InternFeil("Uventet etteroppgjør")` eller noe i den dur for å ha bedre dokumentert feilsituasjonen